distribution/lib/Standard/Base/0.0.0-dev/src/Data/Array.enso

```diff
@@ -286,7 +195,12 @@ type Array
          If the index is less than 0, the index will be counted back from the
          end.
     remove : Integer -> Vector
-    remove self at=-1 = Vector.remove self at
+    remove self at=-1 = (Vector.from_polyglot_array self).remove at
+
+    ## PRIVATE
+    remove_builtin self at =
+        _ = [at]
+        Error.throw (Unimplemented.Error "Array.remove_builtin.")
```

Comment:
Ideally, I think we should filter functions whose doc contains `PRIVATE` in the `VectorArraySignatureTest` and ignore them.

But I'm not sure how hard it is to do that (cc: @JaroslavTulach)

Still, I think worth adding at least a small comment inside of this function (not a doc comment obviously) explaining why it has to exist here.